weaver/tests/network-setups/corda/.gitignore

```diff
@@ -3,6 +3,7 @@ shared/artifacts/*.jar
 shared/artifacts/*.zip
 shared/artifacts/clients-0.1
 shared/artifacts/clients-0.2
+!shared/**/logs
```

Comment:
We want to retain `logs` for checkin? How exactly is this pattern working?

Reply:
We just want to retain the empty directory, not the log files itself.